healthcare/dicomweb_study_retrieve.go

```diff
@@ -50,10 +50,15 @@ func dicomWebRetrieveStudy(w io.Writer, projectID, location, datasetID, dicomSto
 		return fmt.Errorf("could not read response: %v", err)
 	}
 
-	if resp.StatusCode > 299 {
-		return fmt.Errorf("SearchForInstances: status %d %s: %s", resp.StatusCode, resp.Status, respBytes)
+	if err = ioutil.WriteFile(outputFile, respBytes, 0644); err != nil {
```

Comment:
I'm generally hesitant to do IO -- do we need to write the file out? Was there confusion about what to do with the bytes?

Reply:
Simply writing out the bytes is not particularly useful and doesn't demonstrate what users would actually do. Our other samples all write the bytes out to a file so it would also be good to be consistent with those.

Reply:
SGTM. Thanks.